arm-apimanagement/2015-09-15/swagger/service.json

```diff
@@ -0,0 +1,1288 @@
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "type": "file"
+            },
+            "x-nullable": true
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/updatehostname/operationresults/{operationId}": {
+      "get": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_GetApiManagementServiceUpdateHostNameOperationStatus",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "type": "file"
+            },
+            "x-nullable": true
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ApiManagement/operations": {
+      "get": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_GetAllSupportedOperations",
+        "parameters": [ ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}": {
+      "get": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_Get",
+        "description": "Get ApiManagement service (resource description).",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/APIManagementService"
+            }
+          },
+          "404": {
+            "description": "APIManagement service Not Found",
+            "schema": {
+            }
```

Comment:
missing schema for 404.
Defining a negative response code in swagger means that the Autorest generated code will not throw an exception for that code, but will deserialize as per the schema defined for that response. Are you sure you want to do that?

Please take a look at this document to understand how Autorest processes responses (positive. negative, default)
https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#responses